test/functional/page_objects/common_page.ts

```diff
@@ -403,7 +403,11 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
     async closeToastIfExists() {
       const toastShown = await find.existsByCssSelector('.euiToast');
       if (toastShown) {
-        await this.closeToast();
+        try {
+          await find.clickByCssSelector('.euiToast__closeButton');
+        } catch (err) {
+          log.debug(`Failed to close toast: ${err}`);
```

Comment:
What type of scenarios is this supposed to be catching? Can we filter for specific error types rather than ignore all errors?

Reply:
I reverted this change, I will try to find a better way.